### ISSUE TITLE
Add a Changelog file.

### ISSUE THREAD

**Issue by VincentLanglet** [Limenius/ReactBundle#47]
I recently saw there is a major update from 4.0 to 5.0, but I don't see any changelog for this.

Could you add a changelog file in the project ? @nacmartin 
A major update means breaking changes. I can update without knowing the impact.

**Comment by nacmartin** [comment]
There we go https://github.com/Limenius/ReactBundle/releases/tag/v5.0.0

**Comment by VincentLanglet** [comment]
@nacmartin Thanks !
Was the drop of twig 2 necessary ?
A lot of project still use twig 2 and both 
```
use Twig\Extension\AbstractExtension;
use Twig\TwigFunction;
```
are available since something like Twig 2.7 (https://github.com/twigphp/Twig/tree/v2.7.0)
